grails-app/i18n/messages.properties

```diff
@@ -1331,6 +1331,7 @@ orderItem.totalPrice.label=Total Amount
 orderItem.originBinLocation.label=Origin Bin
 orderItem.destinationBinLocation.label=Destination Bin
 orderItem.totalCost.label=Total Cost
+orderItem.recipient.label=Recipient
```

Comment:
Side note: I see tons of same lavels (for example Recipient for shipment, requisition, order, now orderItem etc), perhaps we could do a "small" refactor and get rid of redundant labels and add more universal label properties?

Reply:
Yes, I generally try to use ```xyz.label``` for first-class domains and ```default.xyz.label``` for any properties that are used across domains. But I couldn't find a default for recipient so I added it to order item for now ... with the thought that we'd refactor at some point. The one caveat is that if we have a label for each, then that means people can add their own translation for each domain. For example, someone might want to set orderItem.recipient.label = Recipient while they want to call shipmentItem.recipient.label = Beneficiary. That's a bad example, but at least this would be customizable at every level.

The ideal would be if properties file supported inheritance. Something we could look into, but I assume that's not possible.
```
orderItem.recipient.label = default.recipient.label
shipmentItem.recipient.label = default.recipient.label
```